internal/InfrastructureManagerConfig.go

```diff
@@ -0,0 +1,74 @@
+package internal
```

Comment:
I would suggest to create a separate folder for package containing configuration (e.g. `/internal/config`)